app/src/main/res/animator/channel_item_animator.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item>
+        <set>
+            <objectAnimator
+                android:duration="@android:integer/config_shortAnimTime"
```

Comment:
This is personal opinion, but I feel like this animation might be a little bit too long.  200 is normally a good number, but I wonder if we might want to use 150 or even 100 here.  I do think it's good to err on the side of quicker when it comes to animations.